### PR TITLE
doc: v12.0.2 (dev) release notes

### PR DESCRIPTION
**Pull request by theanalyst**
12.0.2 release notes upto `27c3784`, added all pending changes as per  the PendingReleaseNotes, and a small section on RGW server side encryption

### CODE REVIEW

**Comment by theanalyst** [comment]
need to add a few more commits to this as we had to merge 12.0.1 tag in master for a clean ff

**Comment by theanalyst** [comment]
updated with commits reflecting changes upto 12.0.2

**Comment by theanalyst** [comment]
test this please

File: doc/release-notes.rst [hunk 1]
```diff
@@ -2,6 +2,389 @@
...
+  than it thinks you wanted.  By default this will be 1 if you ever had
+  a standby, and 0 if you did not.  You can customize this using
+  ``ceph fs set <fs> standby_count_wanted <number>``.  Setting it
+  to zero will effectively disable the health check.
+* The "ceph mds tell ..." command has been removed.  It is superceded
+  by "ceph tell mds.<id> ..."
+* RGW introduces server side encryption of uploaded objects with 3 options for
+  the management of encryption keys, automatic encryption (only recommended for
+  test setups), customer provided keys similar to Amazon SSE KMS specification &
+  using a key management service (openstack barbician)
```
Comment (smithfarm) [inline doc/release-notes.rst]: (this feature is supported on x86_64 only)

Comment (theanalyst) [inline doc/release-notes.rst]: @cbodley can you confirm, since isa-l builds only on intel

Comment (cbodley) [inline doc/release-notes.rst]: isa-l is just an optimization, rgw encryption works with or without the plugin

Comment (cbodley) [inline doc/release-notes.rst]: > customer provided keys similar to Amazon SSE KMS specification & using a key management service (openstack barbician)

customer provided keys = amazon SSE-C
key management with barbican = amazon SSE-KMS

**Pull request merged**